src/engine/plugins/table.cpp

```diff
@@ -127,6 +128,12 @@ Status TablePlugin::HandleRequest(const RoutingAlgorithmsInterface &algorithms,
                         result_tables_pair.second[table_index] = distance_estimate;
                     }
                 }
+                if (params.scale_factor > 1.0 &&
+                    result_tables_pair.first[table_index] != MAXIMAL_EDGE_DURATION)
```

Comment:
Hmm should scale_factor also scale the fallback speed result?

Reply:
The usecase for fallback_speed is a guestimate when there's no route. I'd imagine the solver would want realistic estimates for even the fallback to make realistic routes.

Reply:
Yes fallback speeds should mirror real speeds. Scaling one and not the other is questionable when feeding into a optimzation solver.